### PR TITLE
mark all notifications as read

### PR DESCRIPTION
**Pull request by Asna-A**

### CODE REVIEW

**Review by Narayana221 — APPROVED** [review]

File: src/services/WolfDen.Application/DTOs/Attendence/NotificationDTO.cs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 {
     public class NotificationDTO
     {
+        public int NotificationId { get; set; }
         public string Message { get; set; }
```
Comment (muhammedashikCQ) [inline src/services/WolfDen.Application/DTOs/Attendence/NotificationDTO.cs:6]: public string Message { get; set; } = string.Empty;

File: src/services/WolfDen.Application/Requests/Commands/Attendence/UpdateAllNotification/UpdateAllNotificationCommandAHandler.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+namespace WolfDen.Application.Requests.Commands.Attendence.UpdateAllNotification
+{
+    public class UpdateAllNotificationCommandHandler(WolfDenContext context) : IRequestHandler<UpdateAllNotificationCommand,bool>
+    {
+        private readonly WolfDenContext _context = context;
+        public async Task<bool> Handle(UpdateAllNotificationCommand request, CancellationToken cancellationToken)
+        {
+            List<Notification> notifications = await _context.Notifications
+                .Where(x => x.EmployeeId == request.EmployeeId && x.IsRead==false).ToListAsync(cancellationToken);
```
Comment (muhammedashikCQ) [inline src/services/WolfDen.Application/Requests/Commands/Attendence/UpdateAllNotification/UpdateAllNotificationCommandAHandler.cs:14]: format

File: src/services/WolfDen.Domain/Entity/DailyAttendence.cs [hunk 3]
```diff
@@ -32,5 +32,9 @@ public DailyAttendence(int employeeId, DateOnly date, DateTimeOffset arrivalTime
             EmailSent = emailSent;
```
Comment (muhammedashikCQ) [inline src/services/WolfDen.Domain/Entity/DailyAttendence.cs:32]: remove

File: src/services/WolfDen.Application/Requests/Queries/Notifications/GetNotification/GetNotificationQueryHandler.cs [hunk 4]
```diff
@@ -14,6 +14,7 @@ public async Task<List<NotificationDTO>> Handle(GetNotificationQuery request, Ca
                 .Where(x => x.EmployeeId == request.EmployeeId && x.IsRead == false)
```
Comment (muhammedashikCQ) [inline src/services/WolfDen.Application/Requests/Queries/Notifications/GetNotification/GetNotificationQueryHandler.cs:14]: !x.IsRead

File: src/services/WolfDen.Application/DTOs/Attendence/NotificationDTO.cs [hunk 5]
```diff
@@ -2,6 +2,7 @@
...
         public DateTime CreatedAt { get; set; }
```
Comment (muhammedashikCQ) [inline src/services/WolfDen.Application/DTOs/Attendence/NotificationDTO.cs:7]: Rename as CreatedOn

File: src/services/WolfDen.Application/Requests/Commands/Attendence/SendEmail/SendEmailCommandHandler.cs [hunk 6]
```diff
@@ -39,7 +39,7 @@ public async Task<bool> Handle(SendEmailCommand request, CancellationToken cance
             {
                 DailyAttendence? attendence = await _context.DailyAttendence
                .Where(a => a.Date == DateOnly.FromDateTime(DateTime.UtcNow).AddDays(-1) && a.EmployeeId == employee.Id).FirstOrDefaultAsync(cancellationToken);
-                attendence.EmailSent = true;
+                attendence.Update();
```
Comment (muhammedashikCQ) [inline src/services/WolfDen.Application/Requests/Commands/Attendence/SendEmail/SendEmailCommandHandler.cs:42]: attendence may be nullable here.Right?

File: src/services/WolfDen.Application/Requests/Commands/Attendence/UpdateAllNotification/UpdateAllNotificationCommandAHandler.cs (hunk 2 above)
Comment (muhammedashikCQ) [inline src/services/WolfDen.Application/Requests/Commands/Attendence/UpdateAllNotification/UpdateAllNotificationCommandAHandler.cs:14]: !x.IsRead

File: src/services/WolfDen.Application/Requests/Queries/Notifications/GetNotification/GetNotificationQueryHandler.cs [hunk 7]
```diff
@@ -14,6 +14,7 @@ public async Task<List<NotificationDTO>> Handle(GetNotificationQuery request, Ca
...
                 .Select(x=>new NotificationDTO
                 {
+                    NotificationId=x.Id,
```
Comment (muhammedashikCQ) [inline src/services/WolfDen.Application/Requests/Queries/Notifications/GetNotification/GetNotificationQueryHandler.cs:17]: format all the files

**Pull request merged**